SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -47,6 +61,21 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 			const FRepLayoutCmd& Cmd = Changes.RepLayout.Cmds[HandleIterator.CmdIndex];
 			const FRepParentCmd& Parent = Changes.RepLayout.Parents[Cmd.ParentIndex];
 
+#if TRACE_LIB_ACTIVE
+			if (LatencyTracer != nullptr)
+			{
+				TraceKey TraceId = LatencyTracer->RetrievePendingTrace(Object, Cmd.Property);
+				if (TraceId == USpatialLatencyTracer::InvalidTraceKey)
+				{
+					// Possibly sending a nested property? 
+					TraceId = LatencyTracer->RetrievePendingTrace(Object, Parent.Property);
+				}
+				if (TraceId != USpatialLatencyTracer::InvalidTraceKey)
+				{
+					*OutLatencyTraceId = TraceId;
```

Comment:
There's no deference checking on this pointer - should probably be a reference then.

Reply:
Good spot, there should be one!